_meta/beat.yml

```
@@ -245,6 +245,10 @@ apm-server:
     # "Content-Encoding", and "Accept"
     #allow_headers : []
 
+    # Custom headers to return in intake responses, eg. maybe for compliance
```

```suggestion
    # Custom HTTP headers to add to RUM responses, e.g. for security policy compliance.
```

I think it would be helpful to remind readers that it's RUM-specific (even though it's under the "rum" object). Also, using the word "add" helps document that these are additive, rather than replacing existing headers.

Added "HTTP" to align with the "Custom HTTP headers" wording used in the top-level "headers" config

Other suggestions are nits, feel free to disagree.